addons/hw_drivers/main.py

```diff
@@ -13,6 +13,13 @@
 
 _logger = logging.getLogger(__name__)
 
+if platform.system() == 'Windows':
+    try:
+        import schedule
+    except ImportError:
+        schedule = None
+        _logger.error('Could not import library schedule')
```

Comment:
Change to _logger.warning basically it is not an error

Reply:
why wouldn't it be ? 5 lines below for the import of `DBusGMainLoop` it is also logged as an error. DO you want me to change this line too ?